ref-exchange/tests/test_migrate.rs

```diff
@@ -46,7 +46,7 @@ fn test_upgrade() {
     .assert_success();
     let metadata = get_metadata(&pool);
     // println!("{:#?}", metadata);
-    assert_eq!(metadata.version, "1.4.0".to_string());
+    assert_eq!(metadata.version, "1.4.1".to_string());
```

Comment:
It seems the actual version string needs to be updated for version upgrade, looks not necessary to test the version number itself though it's OK.

Reply:
Yes, this test is to ensure the release binary file has the expected version.